sdk/storage/Azure.Storage.Files.Shares/src/Models/ShareProperties.cs

```diff
@@ -119,6 +119,24 @@ public class ShareProperties
         /// </summary>
         public bool? EnableSnapshotVirtualDirectoryAccess { get; internal set; }
 
+        /// <summary>
+        ///  Optional. Supported in version 2023-11-03 and above.  Only applicable for premium file storage accounts.
```

Comment:
NIT - Is this supposed to be 2024-11-04 ? 

Same for the rest of the comments below

Reply:
Fixed.